test/commands/convert/hdf2ckl.test.ts

```diff
@@ -9,10 +9,10 @@ describe('Test hdf2checklist', () => {
 
   test
     .stdout()
-    .command(['convert hdf2ckl', '-i', path.resolve('./test/sample_data/HDF/input/red_hat_good.json'), '-o', `${tmpobj.name}/hdf2ckl_test.json`])
+    .command(['convert hdf2ckl', '-i', path.resolve('./test/sample_data/HDF/input/vSphere8_report.json'), '-o', `${tmpobj.name}/hdf2ckl_test.json`])
```

Comment:
Should we add this as a test rather than remove the previous? Or remove the other file if it is no longer needed?

Reply:
The red_hat file is still being used in the test with metadata. But yeah, we should probably add it as a test for profile with dependents.